tests/base64-raw.rs

```diff
@@ -19,5 +19,5 @@ pub static DATA: &str = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/asset
 
 #[test]
 fn test() {
-    verify_str("base64.txt", &DATA);
+    verify_str("base64.txt", DATA);
```

Comment:
This change is unrelated to the proc macro hack stuff, just normal code cleanup right?

Reply:
yes, it was the only clippy warn I was getting so I just cleaned it up rq